### DISCUSSION TITLE
Baryon Acoustic Oscillations

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#289] [Astrophysics and Physics]
New paper from DESI that augments the DR1 with "full shape" cluster data.
https://arxiv.org/abs/2411.12022v1
companion papers:
https://data.desi.lbl.gov/doc/papers/
https://drive.google.com/file/d/1za7cDEget8u39peYXTGOCLX0bOgA1aFZ/view
Combined analysis leads to record constraints:
> 1% precision in Ωm, 0.6% in σ8 and S8, and 0.4% precision
in H0.

This increase the Hubble tension to ?? sigmas
Probably increase the S8 tension
Also evidence for "evolving" dark energy is similar.

Constraints on modified gravity: significance unclear, seems to strenghten the case for GR.

also possible future neutrino anomaly
Previous neutrino experiments found that the sum of the masses of the three types of neutrinos should be at least 0.059 eV/c2. (For comparison, an electron has a mass of about 511,000 eV/c2.) DESI's results indicate that the sum should be less than 0.071 eV/c2, leaving a narrow window for neutrino masses.
though counterpoints about previous study:
https://www.google.com/search?q=desi+dr1+neutrino+mass&rlz=1C1VDKB_frFR1053FR1053&oq=desi+dr1+neutrino+mass&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIHCAEQIRigATIHCAIQIRigAdIBCDYzMDNqMGo3qAIAsAIA&sourceid=chrome&ie=UTF-8

**Comment by RedshiftDrift** [comment]
@LifeIsStrange there's also a [Cosmology Talk](https://youtu.be/-2mlU-YzEbw) about this work, featuring three astronomers from DESI.

I didn't have 110 minutes to spare on the video, but it seems they get more or less the same $\Omega_m$, $\sigma_8$, and $H_0$ already given by $\Lambda CDM$.

$\ $
>  evidence for "evolving" dark energy is similar.

I'd say it's evidence for an additional epicycle: ***"evolving" dark energy***

**Comment by LifeIsStrange** [comment]
https://arxiv.org/abs/2411.17623

**Reply by RedshiftDrift** [reply]
👍 The epicycle even has a name: "the primordial non-Gaussianity parameter $f_{NL}^{loc}$"
